invenio_app_ils/records/jsonresolver/document.py

```diff
@@ -27,36 +24,23 @@ def jsonresolver_loader(url_map):
     def circulation_resolver(document_pid):
         """Return circulation info for the given Document."""
         circulation = {}
-        active_loans_count = 0
-        loanable_items_count = 0
-        items_count = 0
-        item_pids = circulation_items_retriever(document_pid)
-
-        for pid in item_pids:
-            items_count += 1
-            if not is_item_available(pid):
-                active_loans_count += 1
 
-            if circulation_is_item_available(pid):
-                loanable_items_count += 1
+        items = list(ItemSearch().search_by_document_pid(document_pid).scan())
+        items_count = len(items)
```

Comment:
`items_count` and `loanable_items_count` can be directly assigned to the relevant circulation property, to avoid the clatter :)